eslint.config.mjs

```diff
@@ -123,6 +125,9 @@ export default tseslint.config([
 
       'no-floating-promise/no-floating-promise': 'error',
 
+      // Use the recommended rules for react-you-might-not-need-an-effect
+      ...reactYouMightNotNeedAnEffect.configs.recommended.rules,
```

Comment:
At least per the project's README, it looks like these are all warnings by default. My personal opinion: lint rules that warn are essentially useless because they don't block anything in CI. IMO if a rule is important, it should error, otherwise it shouldn't exist.

Reply:
I'll elevate these to errors.